clusterloader2/pkg/framework/framework.go

```diff
@@ -33,14 +33,10 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
-const (
-	// AutomanagedNamespaceName is a basename for automanaged namespaces.
-	AutomanagedNamespaceName = "namespace"
-)
-
 // Framework allows for interacting with Kubernetes cluster via
 // official Kubernetes client.
 type Framework struct {
+	automanagedNamespaceName  string
```

Comment:
automanagedNamespacePrefix ?

in the end, names have "-number"
https://github.com/kubernetes/perf-tests/blob/40503f822c514584dad1306da3494234221d13c4/clusterloader2/pkg/framework/framework.go#L81

Reply:
Done.